Assets/Scripts/Character.cs

```diff
@@ -0,0 +1,10 @@
+﻿using UnityEngine;
+using ARPGTD.CharacterStats;
+
+public class Character : MonoBehaviour
+{
```

Comment:
We need to pick a way to set up brackets and stick with it. I like having brackets on the same line as the class declaration 
ie: `public class Character : MonoBehaviour { ...`